Header header
float64 goal_x
float64 goal_y
float64[] obs_x
float64[] obs_y
